rigelprotocol/SmartswapDappV2
# shows user history transactions

#### What does this PR do?
-Shows user swap transaction history on the app. 

#### What has been completed?
- Transaction history tab shows dynamic data based on user swap transaction history 

#### How should the changes be manually tested?
- Connect a wallet and view the transaction history tab to see your previous transactions. 

#### Demo Video and screenshots?
-Loom : https://www.loom.com/share/20250bdf891848dbb0e6d2b315485485
